python/kwiver/vital/tests/CMakeLists.txt

```diff
@@ -43,9 +43,20 @@ set(__all_modnames
   ${__testable_modnames}
   )
 
-# Add files to the vital.tests python module
+
+set(mod_dst)
```

Comment:
I'm sorry I don't see it right away, but what is this doing?

Reply:
That is a variable I missed when I was doing cleanup. Removed.